hooks/pre/31-mongo_storage_engine.rb

```diff
@@ -0,0 +1,51 @@
+if answers['katello']
+  MONGO_DB_NAME = param('katello', 'pulp_db_name')
+else
+  MONGO_DB_NAME = param('foreman_proxy_content', 'pulp_db_name')
+end
+
+def migration
+  export_dir = '/var/tmp/engine_upgrade'
+  mongo_dir = '/var/lib/mongodb'
+  hiera_file = '/etc/foreman-installer/custom-hiera.yaml'
+  mongo_backup = '/var/tmp/mongo_backup'
+
+  # Create export directory and dump MongoDB
+  logger.info 'Stopping services except MongoDB'
+  Kafo::Helpers.execute('katello-service stop --exclude rh-mongodb34-mongod')
```

Comment:
re https://github.com/Katello/katello-installer/pull/639#discussion_r193008812

what is the service name `mongo-syspaths` installs? I dont know, but if it does what i understand it does, then surely the service would be called `mongod`?

Reply:
Sorry I was incorrect, the syspaths just make it so we dont need to use scl enable mongo-3.4-blah  bash. This service name is what is currently in katello-service and Foreman Maintain so I think we should just keep it like it is, since its in the upgrade hook already for the other pulp/mongo stuff.